packages/nuxi/src/commands/init.ts

```diff
@@ -41,22 +46,60 @@ export default defineNuxtCommand({
       process.exit(1)
     }
 
-    // Show next steps
-    const relativeDist = rpath(t.dir)
+    // Prompt user to select package manager
+    const selectedPackageManager = (await consola.prompt(
+      'Which package manager would you like to use?',
+      {
+        type: 'select',
+        options: ['npm', 'pnpm', 'yarn']
+      }
+    )) as unknown as PackageManagerName
+
+    // Get relative project path
+    const relativeProjectPath = relative(process.cwd(), t.dir)
 
     // Write .nuxtrc with `shamefully-hoist=true` for pnpm
-    const usingPnpm = (process.env.npm_config_user_agent || '').includes('pnpm')
-    if (usingPnpm) {
-      await writeFile(`${relativeDist}/.npmrc`, 'shamefully-hoist=true')
+    if (selectedPackageManager === 'pnpm') {
+      await writeFile(`${relativeProjectPath}/.npmrc`, 'shamefully-hoist=true')
+    }
```

Comment:
1. The comment specifices a different filename to what is in the actual code. Shall it be 'Write .npmrc with (...)'?
2. Knowing the `.npmrc` file is already in the default template, wouldn't its contents be overwritten?